### PR TITLE
Noindexing for series pages

### PR DESCRIPTION
**Pull request by ColinBarnwell**
This builds on a previous spike in order to add `noindex` tagging for some series pages. 

Note: As part of the kick off we discussed combining the non-index brand lists into a shared object, but from a product point of view it is preferred if we keep them separate.

**North West Tonight**
- [x] /programmes/b006pfjx/episodes/guide
- [x] /programmes/b006pfjx/episodes/player
- [x] /programmes/b006pfjx/broadcasts/upcoming

**Match of the Day**
- [x] /programmes/b007t9y1/episodes/guide
- [x] /programmes/b007t9y1/episodes/player
- [x] /programmes/b007t9y1/broadcasts/upcoming

**Line of Duty**
- [x] /programmes/p00yzlr0/episodes/guide
- [x] /programmes/p00yzlr0/episodes/player
- [x] /programmes/p00yzlr0/broadcasts/upcoming
- [x] /programmes/b01k9pm3
- [x] /programmes/b03vp2vy
- [x] /programmes/b07500bw
- [x] /programmes/b08l5tmp
- [x] /programmes/m0003v98
- [x] /programmes/m000tjg9

**Fleabag**
- [x] /programmes/p070npjv/episodes/guide
- [x] /programmes/p070npjv/episodes/player
- [x] /programmes/p070npjv/broadcasts/upcoming
- [x] /programmes/p040tlqx
- [x] /programmes/p071bjr7

**Saturday Kitchen**
- [x] /programmes/b006v5y2/episodes/guide
- [x] /programmes/b006v5y2/episodes/player
- [x] /programmes/b006v5y2/broadcasts/upcoming

**BBC News**
- [x] /programmes/b006mgyl/episodes/guide
- [x] /programmes/b006mgyl/episodes/player
- [x] /programmes/b006mgyl/broadcasts/upcoming

**Powering Britain**
- [x] /programmes/m000lxp1/episodes/guide
- [x] /programmes/m000lxp1/episodes/player
- [x] /programmes/m000lxp1/broadcasts/upcoming

**Impossible**
- [x] /programmes/b08s3bgz/episodes/guide
- [x] /programmes/b08s3bgz/episodes/player
- [x] /programmes/b08s3bgz/broadcasts/upcoming
- [x] /programmes/b087zc2h
- [x] /programmes/b08s2vj2
- [x] /programmes/b09bbp4y
- [x] /programmes/b0bdmdp9
- [x] /programmes/b0bqrnmv
- [x] /programmes/m0004hc4
- [x] /programmes/m000npn9
- [x] /programmes/m000t93p

## Jira

https://jira.dev.bbc.co.uk/browse/DATCAP-232

### CODE REVIEW

**Review by murkas — APPROVED** [review]

File: src/Controller/FindByPid/SeriesController.php [hunk 1]
```diff
@@ -38,6 +38,14 @@ public function __invoke(
         Breadcrumbs $breadcrumbs
     ) {
         $this->setAtiContentLabels('series', 'series');
+        $noIndexBrands = [
+            'p00yzlr0', // Line of Duty
+            'p070npjv', // Fleabag
+            'b08s3bgz',  // Impossible
+        ];
+        if (in_array($programme->getTleo()->getPid(), $noIndexBrands)) {
```
Comment (spacekitcat) [inline src/Controller/FindByPid/SeriesController.php:46]: Is there a risk of null reference dereferencing here? (ditto the rest)

**Review by spacekitcat — APPROVED** [review]
Looks good, just one code safety releated query.

File: src/Controller/FindByPid/SeriesController.php (hunk 1 above)
Comment (ColinBarnwell) [reply src/Controller/FindByPid/SeriesController.php:46]: Let me know if I've missed anything:
`programme` is always present -- the controller cannot work without it.
`getTleo()` will return either the top level ancestor of the programme, or the programme itself if i has none.
`getPid()` cannot return a null as it is the required ID field of the entity.

Comment (spacekitcat) [reply src/Controller/FindByPid/SeriesController.php:46]: That's fair, I don't know anything about the internals of those methods!

**Review by ajroberts10 — APPROVED** [review]
nice one 👍🏻

**Review by llh1 — APPROVED** [review]

**Pull request merged**